frontend/uddhar-frontend/src/shared/components/Navbar.jsx

```
@@ -43,10 +49,10 @@ const Navbar = ({  children  }) => {
   const signIn = (
     <>
       <p className="text-lg hover:text-gray-500">
-        <button className="cursor-pointer" onClick={() => navigate("/sign-in")}>Sign In</button>
+        <button className="cursor-pointer" onClick={() => {navigate("/sign-in"); setShowMenu(false); setUserMenuOpen(false); }}>Sign In</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)